### PR TITLE
Update README.md in ChromeExtension

### PR DESCRIPTION
**Pull request by Karlie-777**

### CODE REVIEW

File: tools/chrome-debug-extension/README.md [hunk 1]
```diff
@@ -44,4 +44,4 @@ When you load a configuration, the contents of the configuration will be shown o
 
 ## Contributing to the tool
 
-This tool is internally open sourced - if you would like to make a contribution, `INSERT_PROCESS_HERE`
+This tool is internally open sourced - if you would like to make a contribution, please follow the process [here](https://github.com/microsoft/ApplicationInsights-JS/tree/ChromeDebugExtension#contributing)
```
Comment (MSNev) [inline tools/chrome-debug-extension/README.md:47]: The ChromeDebugExtension will be removed once we push this into master, so this should point to the master version

File: tools/chrome-debug-extension/README.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ Since each web application logs telemetry in a different schema, you'll first be
 
 You can draft a configuration directly in the tool on the `Configuration Selection` page. You can always get back to that page by clicking the "gear" icon.
 
-To add your configuration to the dropdown list, you'll need to check in a change to `INSERT_PATH_TO_FILE` to add your configuration's URL to the list. Follow the instructions in that file.
+To add your configuration to the dropdown list, you'll need to check in a change to [`src/configuration/Configuration.types.ts`](https://github.com/microsoft/ApplicationInsights-JS/blob/a9c3948db6d7dddf58cc6450735958667a79d0ac/tools/chrome-debug-extension/src/configuration/Configuration.types.ts#L16) to add your configuration's URL to the list. Follow the instructions in that file.
```
Comment (MSNev) [inline tools/chrome-debug-extension/README.md:35]: We will eventually need to update this to point to the master version

File: tools/chrome-debug-extension/README.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ Your browser will load the latest version of the tool each time you restart the
 
 ## How to run the tool
 
-Click the `INSERT_ICON_HERE` icon next to the address bar to launch the tool.
+Click the `small puzzle piece` icon next to the address bar and select `Telemetry Viewer`![Telemetry-Viewer-icon](https://github.com/microsoft/ApplicationInsights-JS/blob/ChromeDebugExtension/tools/chrome-debug-extension/images/icon-19.png) from the extension dropdown to launch the tool.
```
Comment (MSNev) [inline tools/chrome-debug-extension/README.md:25]: We will eventually need to update this to point to the master version as ChromeDebugExtension branch will be removed once we merge

File: tools/chrome-debug-extension/README.md (hunk 2 above)
Comment (Karlie-777) [reply tools/chrome-debug-extension/README.md]: updated to relative path

File: tools/chrome-debug-extension/README.md (hunk 3 above)
Comment (Karlie-777) [reply tools/chrome-debug-extension/README.md]: updated to relative path as well

**Comment by Karlie-777** [comment]
should add icon image to the `chrome extension puzzle icon` after the icon image is merged

**Review by MSNev — APPROVED** [review]

**Pull request merged**